account_credit_control/models/credit_control_run.py

```diff
@@ -100,17 +100,18 @@ def _check_run_date(self, controlling_date):
         )
         if runs:
             raise UserError(
-                _('A run has already been executed more '
-                  'recently than %s') % runs.date)
+                _('A run has already been executed more recently (%s)')
+                % (runs.date)
```

Comment:
```suggestion
                % (runs.date,)
```

The comma is important for a 1-uple, as `(x)` is the same as `x`.

Reply:
But since `runs.date` is never a tuple nor a list, it does not matter.